lib/scenario_runner.py

```diff
@@ -818,50 +818,62 @@ def _build_docker_images(self):
                     ps = subprocess.run(docker_build_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='UTF-8', errors='replace', check=False)
 
                 if ps.returncode != 0:
-                    print(f"Error: {ps.stderr} \n {ps.stdout}")
-                    raise OSError(f"Docker build failed\nStderr: {ps.stderr}\nStdout: {ps.stdout}")
+                    raise subprocess.CalledProcessError(ps.returncode, 'Docker build failed', output=ps.stdout, stderr=ps.stderr)
```

Comment:
Why are you calling one `output` and one `stderr`. Wouldn't either `stdout` or `error` be the consistent wording?